fix scale and zero_point names

dskhudia opened the pull request:
Summary: scale and zero_point name should match with what's used in other methods of the class.

Closes #23881

Differential Revision: D16703956

jianyuh reviewed (approved):
LGTM! Please land after it passes all OSS tests

jerryzh168 reviewed (approved):
Thanks, is there any tests that you need to change?

dskhudia commented:
All tests are passing.

dskhudia committed:
fix scale and zero_point names (#23991)

Summary:
scale and zero_point name should match with what's used in other methods of the class.

Closes https://github.com/pytorch/pytorch/issues/23881
Pull Request resolved: https://github.com/pytorch/pytorch/pull/23991

Test Plan: buck build mode/opt caffe2/benchmarks/operator_benchmark/pt:qconv_test --show-output

Reviewed By: jianyuh

Differential Revision: D16703956

Pulled By: dskhudia

fbshipit-source-id: 9f0fea383adebea981f676f0b49e76a779d7c604
